app/controllers/api/movies_controller.rb

```diff
@@ -0,0 +1,11 @@
+class Api::MoviesController < ApplicationController
+  require 'rest-client'
```

Comment:
Do you need to require `rest-client`? In that case, please put it with the other requires in your `application.rb` 👍🏻

Reply:
in documentation they were requiring it, but it seems to work without require, so we removed it.